app/src/main/java/com/helio/app/ui/sensors/SensorsSettingsFragment.java

```diff
@@ -0,0 +1,70 @@
+package com.helio.app.ui.sensors;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.helio.app.R;
+import com.helio.app.UserDataViewModel;
+import com.helio.app.model.Sensor;
+
+
+public class SensorsSettingsFragment extends Fragment {
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_sensors_settings, container, false);
+        UserDataViewModel model = new ViewModelProvider(requireActivity()).get(UserDataViewModel.class);
+        // Currently I didn't know which sensor will be chosen.
+        Sensor sensor = null;
+        SensorsRecViewAdapter adapter = new SensorsRecViewAdapter(getContext(), model,sensor);
+
+        // Insert into the recycler view
+        RecyclerView recView = view.findViewById(R.id.sensorsRCView);
+        recView.setAdapter(adapter);
+        recView.setLayoutManager(new LinearLayoutManager(getContext()));
+        return view;
+    }
+
+
+
+//    // Adam Sorry I just note your code because currently I am using different method to write about this part.from Jeremy
```

Comment:
This is not very important, but the `main` branch shouldn't have large commented sections of code.
I wrote a comment about this under `fragment_single_sensor_settings.xml` about how we should probably follow how we did it for the single blind settings, to keep things consistent.

Reply:
Don't worry about this though, as @adamb56789 will be working on the functionality.